src/Frontend-GNOME/Views/Chats/ChatView.cs

```diff
@@ -519,6 +520,8 @@ public virtual void Sync()
 
             // REMOTING CALL
             SyncedLastSeenHighlight = _ChatModel.LastSeenHighlight;
+            // REMOTING CALL
+            SyncedLastSeenMessage = _ChatModel.LastSeenMessage;
```

Comment:
Will do. You mentioned that merely checking for >= 0.12 will break dev builds. Should I bump the version to 0.12.1 then? (And if so, where can I do it? :) )